### ISSUE TITLE
ADAM model from R

### ISSUE THREAD

**Issue by ncooder** [Nixtla/statsforecast#886]
### Description

Are you going to add other models than AutoCES from smooth package like ADAM:
[here](https://cran.r-project.org/web/packages/smooth/smooth.pdf)

**Comment by MMenchero** [comment]
Hi @ncooder, 
Are there any specific models you would like to see included in StatsForecast? We are always striving to improve the library, but as you can imagine, adding new models is very time-consuming. So we prefer to prioritize features or models that our users need the most.

**Comment by ncooder** [comment]
Yes, there are more than AutoCES models that we can use in the statsforecast. The ADAM model is one of them from the smooth model that is great and probably no one will implement that in Python besides the Statsforecast. I can help with the development, but still I would need a guidance. I really appreciate the models you have there, but please try to add more SOTA models.

**Comment by MMenchero** [comment]
If you would like to implement the ADAM model, you can start with the [contributing guide](https://github.com/Nixtla/statsforecast/blob/main/CONTRIBUTING.md). 

if you have specific questions, please let us know.

**Comment by marcozanotti** [comment]
Hi @MMenchero , 

related to the request there are not only a few SOTA methods like:

- ADAM (Augmented Dynamic Adaptive Model),  
- SSARIMA (State Space ARIMA) and its automatic version,
- MSARIMA (Multiple Seasonal ARIMA),
- iETS (Intermittent ETS)  

 but also some essential functionalities like external regressors within the ETS method.
 Thank you for all your work.
